ansible.spec

```diff
@@ -0,0 +1,147 @@
+################################################################################
+
+%{!?python_sitelib: %global python_sitelib %(%{__python} -c "from distutils.sysconfig import get_python_lib; print(get_python_lib())")}
+
+%if 0%{?rhel} == 5
+%define __python /usr/bin/python26
+%endif
+
+################################################################################
+
+Summary:              Radically simple IT automation
+Name:                 ansible
+Version:              2.0.1.0
+Release:              0%{?dist}
+URL:                  http://www.ansible.com
+License:              GPLv3
+Group:                Development/Libraries
+
+Source:               http://releases.ansible.com/ansible/%{name}-%{version}.tar.gz
+
+BuildRoot:            %{_tmppath}/%{name}-%{version}-%{release}-buildroot
+
+BuildArch:            noarch
+
+%if 0%{?rhel} && 0%{?rhel} <= 5
+BuildRequires:        python26-devel
+BuildRequires:        python26-setuptools
+
+Requires:             python26-PyYAML
```

Comment:
Don't use different Requires tag for each dependency. I prefer to define several deps in one line.